tests/cypress/cypress/pages/MenuTabs.js

```diff
@@ -0,0 +1,157 @@
+    }
+
+    //Get primary menu items
+    //Get primary menu options
+    get primaryMenuOptions(){
+      cy.get("body").then($body => {
+        if ($body.find("#block-acquia-claro-primary-local-tasks > nav > ul > li.is-active button.reset-appearance").length > 0) {
+          cy.get("#block-acquia-claro-primary-local-tasks > nav > ul > li.is-active button.reset-appearance").then($dropdown => {
+            if ($dropdown.is(':visible')){
+              cy.get("#block-acquia-claro-primary-local-tasks > nav > ul > li.is-active button.reset-appearance").click()
+            }
+          })
+        }
+      })
+    }
+    //Get view primary menu
+    get viewMenu() {
+        return cy.get("#block-tabs-2 > nav > ul > li:nth-child(1) > a")
+    }
+    //Get scheduled primary menu
+    get scheduledMenu() {
+        return cy.get("#block-tabs-2 > nav > ul > li:nth-child(2) > a")
+    }
+    //Get scheduled primary menu
+    get scheduledMediaMenu() {
+        return cy.get("#block-acquia-claro-primary-local-tasks > nav > ul > li:nth-child(3) > a")
+    }
+    //Get edit primary menu
+    get editMenu() {
+        return cy.get("#block-acquia-claro-primary-local-tasks > nav > ul > li:nth-child(4) > a")
+    }
+    //Get moderation dashboard primary menu
+    get moderation_dashboardMenu() {
+        return cy.get("#block-acquia-claro-primary-local-tasks > nav > ul > li:nth-child(5) > a")
+    }
+    //Get Acquia DAM primary menu
+    get acquiaDamMenu() {
+        return cy.get("#block-acquia-claro-primary-local-tasks > nav > ul > li:nth-child(6) > a")
+    }
+    //Get clone primary menu
+    get cloneMenu() {
+        return cy.get("#block-tabs-2 > nav > ul > li:nth-child(7) > a")
+    }
+
+    //Validate the spellsings for Menu items
+    spell_Validations_Menu() {
+        this.homeMenu.should('have.text', 'Home')
+        this.articlesMenu.should('have.text', 'Articles')
+        this.eventMenu.should('have.text', 'Events')
+        this.peopleMenu.should('have.text', 'People')
+        this.placesMenu.should('have.text', 'Places')
+    }
+
+    //Validate menu items are clickable
+    click_menu_items() {
+
+        this.homeMenu.click({
+            force: true
+        })
+
+        cy.url().should('eq', testData.$home_url)
+
+        this.articlesMenu.click({
+            force: true
+        })
+        cy.url().should('eq', testData.$article_url)
+
+        this.eventMenu.click({
+            force: true
+        })
+        cy.url().should('eq', testData.$events_url)
+
+        this.peopleMenu.click({
+            force: true
+        })
+        cy.url().should('eq', testData.$people_url)
+
+        this.placesMenu.click({
+            force: true
+        })
+        cy.url().should('eq', testData.$places_url)
+
+    }
+    spell_validations_primary_menu() {
```

Comment:
Add a small comment to describe what this custom function do

Reply:
done